src/strategies/MangroveOffer.sol

```diff
@@ -201,17 +191,10 @@ abstract contract MangroveOffer is AccessControlled, IOfferLogic {
       amount = MGV.balanceOf(address(this));
     }
     // the require below is necessary if the `receive()` function is overriden
-    require(MGV.withdraw(amount), "mgvOffer/withdrawFromMgv/withdrawFail");
+    require(MGV.withdraw(amount), "mgvOffer/withdrawFail");
     (bool noRevert,) = receiver.call{value: amount}("");
     // if `receiver` is actually not payable
-    require(noRevert, "mgvOffer/withdrawFromMgvFail");
-  }
-
-  /// @inheritdoc IOfferLogic
-  function tokenBalance(IERC20 token, address maker) external view override returns (uint) {
-    AbstractRouter router_ = router();
-    address makerReserve = reserve(maker);
-    return router_ == NO_ROUTER ? token.balanceOf(makerReserve) : router_.reserveBalance(token, makerReserve);
+    require(noRevert, "mgvOffer/weiTransferFail");
   }
```

Comment:
See 7.2.B.
`reserve(maker` is no longer necessarily controlled by `maker`. So we cannot provide a generic way to know what is the balance of a particular offer owner (this is now strat implementation dependent)